src/github.ts

```diff
@@ -43,6 +43,8 @@ export interface GitHubOpts {
   token?: string;
 }
 
+const cloudBuildsURL = 'https://app.docker.com/build/accounts/docker/builds';
```

Comment:
I realize that I have the same mistake in my PR but we have to use a param for the account part (hard-coded to docker here). The account is part of the builder endpoint.

Reply:
Ah ok it's the builder name I assume?

Reply:
I think build and push action creates the builder with a -name flag right? So it might not be in the builder name.

Reply:
Ah yes this is part of the endpoint: https://docs.docker.com/build-cloud/ci/#github-actions

Reply:
Updated, we would just need to pass `driver` and `endpoint` to write summary opts. Let me know what you think.